rule/package_comments.go

```diff
@@ -162,3 +162,11 @@ func (l *lintPackageComments) Visit(_ ast.Node) ast.Visitor {
 	}
 	return nil
 }
+
+func isEmptyDoc(commentGroup *ast.CommentGroup) bool {
+	if commentGroup == nil {
```

Comment:
Don't we have something like that in exported?

Can we reuse the same code?

Reply:
I think so... I've didn't search for it.